### PR TITLE
Replace broken svg

### PR DESCRIPTION
**Pull request by NigelBreslaw**

### CODE REVIEW

File: examples/fancy-switches/DarkModeSwitch.slint [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-        property <length> sizeMultiplyer: 0.7px;
-        i := Image {
-            source: @image-url("images/line2.svg");
-            width: self.source.width * sizeMultiplyer * 0.8;
-            height: self.source.height * sizeMultiplyer;
-            colorize: red;
-        }
+
+    i := Image {
+        property <length> sizeMultiplyer: 0.3px;
```
Comment (tronical) [inline examples/fancy-switches/DarkModeSwitch.slint:11]: Multiplier ;-)

File: examples/fancy-switches/DarkModeSwitch.slint [hunk 2]
```diff
@@ -1,36 +1,36 @@
 // Copyright © SixtyFPS GmbH <info@slint.dev>
 // SPDX-License-Identifier: MIT
 
-component SunRayThick {
+export component RayThick {
```
Comment (tronical) [inline examples/fancy-switches/DarkModeSwitch.slint:4]: Is the export intentional?

**Review by tronical — APPROVED** [review]

File: examples/fancy-switches/DarkModeSwitch.slint (hunk 2 above)
Comment (NigelBreslaw) [reply examples/fancy-switches/DarkModeSwitch.slint]: fixed

File: examples/fancy-switches/DarkModeSwitch.slint (hunk 1 above)
Comment (NigelBreslaw) [reply examples/fancy-switches/DarkModeSwitch.slint]: Whoops. Thx!

**Review by tronical — APPROVED** [review]

**Pull request merged**